src/troute-nwm/src/nwm_routing/__main__.py

```diff
@@ -1115,6 +1171,40 @@ def nwm_route(
     else:
         LOG.info(f"executing routing computation ...")
 
+    if (firstRun):
+        compute_log_mc(
```

Comment:
it throw an error on this line that `NameError: name 'compute_log_mc' is not defined`.
( in yaml file, uncomment `log_directory : Log/` and check it out)